libntcan/package.xml

```diff
@@ -1,19 +1,17 @@
-<package>
+<?xml version="1.0"?>
+<package format="2">
   <name>libntcan</name>
   <version>0.6.1</version>
   <description>This package wraps the libntcan to use it as a ros dependency.</description>
-  <!-- <maintainer>Florian Weisshardt</maintainer> -->
 
   <license>proprietary</license>
 
   <url type="website">http://www.esd-electronics.com</url>
-  <!-- <url type="bugtracker"></url> -->
   <maintainer email="fmw@ipa.fhg.de">Florian Weisshardt</maintainer>
   <author>Florian Weisshardt</author>
+
   <buildtool_depend>catkin</buildtool_depend>
+
   <build_depend>dpkg</build_depend>
```

Comment:
good question. It is used to recover the type of architecture using `FIND_PROGRAM`. `dpkg` can be installed using rosdep install (if this is possible... :-S) as it is defined in the [base.yaml](https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml#L537).

We do not need any headers or something from it, so IMO we don't need to `find_package()` it. And it is also not an `exec_depend` as it is just required during building. Thus, okay like this?